Library/Homebrew/descriptions.rb

```diff
@@ -0,0 +1,167 @@
+require "formula"
+require "csv"
+
+class Descriptions
+
+  CACHE_FILE = HOMEBREW_CACHE + "desc_cache"
+
+  def self.cache
+    @cache || self.load_cache
+  end
+
+  # If the cache file exists, load it into, and return, a hash; otherwise,
+  # return nil.
+
```

Comment:
Mind removing the spaces between comments and functions? Makes it easier to know what function they apply fo.